invenio_github/tasks.py

```diff
@@ -71,37 +70,40 @@ def release_default_exception_handler(release, ex):
 ]
 
 
-@shared_task(max_retries=6, default_retry_delay=10 * 60, rate_limit='100/m')
+@shared_task(max_retries=6, default_retry_delay=10 * 60, rate_limit="100/m")
 def disconnect_github(access_token, repo_hooks):
     """Uninstall webhooks."""
     # Note at this point the remote account and all associated data have
     # already been deleted. The celery task is passed the access_token to make
     # some last cleanup and afterwards delete itself remotely.
-    import github3
 
+    # Local import to avoid circular imports
```

Comment:
this points to a clear design/separation issue 😅 😇

Reply:
Agree, however, this was already there and I didn't want to mess too much with it:

https://github.com/inveniosoftware/invenio-github/blob/master/invenio_github/tasks.py#L82

I think that we need to have a cleanup phase after this is merged ...